src/mempool_args.h

```diff
@@ -10,6 +10,9 @@ namespace kernel {
 struct MemPoolOptions;
 };
 
+/** Default for -persistmempool */
+static constexpr bool DEFAULT_PERSIST_MEMPOOL = true;
```

Comment:
nit in 5b426d71aab1d2c7a3f7b44043e98a0e9b821518:

Could use `{}`.

Reply:
Fixed as of 930f023ea9